crates/db_views_actor/src/comment_reply_view.rs

```diff
@@ -247,12 +249,13 @@ impl CommentReplyView {
   pub async fn get_unread_replies(
     pool: &mut DbPool<'_>,
     my_person_id: PersonId,
+    show_bot_accounts: bool,
```

Comment:
```suggestion
    local_user: &LocalUser,
```

Reply:
that'll complicate the rust test a bit, as that currently only inserts `Person`s but no `LocalUser`.

will have a look tomorrow to insert a `LocalUser` for the recipient there.

Reply:
implemented